app/views/lessons/index.html.erb

```diff
@@ -2,28 +2,37 @@
 <div class="text-center mt-4">
     <h1>Categories List</h1>
 </div>
-
-<div class="col text-center mt-4">
-    <button type="button" class="btn btn-outline-secondary w-25">All</button>
-    <button type="button" class="btn btn-outline-success w-25">Learned / In progress</button>
-    <button type="button" class="btn btn-outline-danger w-25">Not touched yet</button>
-</div>
-
-<div class="row mt-5 mx-auto container">
+<div class="row mt-3 mx-auto container">
+<nav>
+  <div class="nav nav-pills mb-3" id="pills-tab" role="tablist">
+    <a class="nav-item nav-link active" id="nav-home-tab" data-toggle="tab" href="#nav-home" role="tab" aria-controls="nav-home" aria-selected="true">All</a>
+    <a class="nav-item nav-link" id="nav-profile-tab" data-toggle="tab" href="#nav-profile" role="tab" aria-controls="nav-profile" aria-selected="false">Learned / In progress</a>
+    <a class="nav-item nav-link" id="nav-contact-tab" data-toggle="tab" href="#nav-contact" role="tab" aria-controls="nav-contact" aria-selected="false">Not touched yet</a>
```

Comment:
please the following parts of the code:
`href="#nav-home"` to `href="#all"`
`href="#nav-profile"` to `href="#learned"`
`href="#nav-contact"` to `href="#not_touched"`